### ISSUE TITLE
Indentation after Ruby string interpolation

### ISSUE THREAD

**Issue by johaywood** [adobe/brackets#11651]
After a line using Ruby string interpolation that also uses braces or parens, Brackets un-indents the lines below the interpolation. There may be some other behavior at play here, but these are the times I seem to run into it and keep indenting manually after editing each line. 

Brackets would try to indent the following code like this:

```
# test.rb

def foo(hash)
  if !hash.empty?
    hash.each do |x|
      puts "value for key y is #{x[:y]}"
  end
else
  puts "string is not bar"
end
end
```

**Comment by MarcelGerber** [comment]
This has already been fixed in CodeMirror (https://github.com/codemirror/CodeMirror/issues/3365) and thus, it'll be fixed in Brackets Release 1.5

**Comment by johaywood** [comment]
:+1:
